docs/source/concepts/FittingMinimizers.rst

```diff
@@ -0,0 +1,205 @@
+.. _FittingMinimizers:
+
+Comparing Minimizers
+====================
+
+Minimizers play a central role when :ref:`Fitting a model <Fitting>`
+in Mantid. Given the following elements:
+
+- a dataset (e.g. a spectrum),
+- a model or function to fit (e.g. a peak or background function, with parameters),
+- an initial guess or starting point for the parameters of the function,
+- a cost function (e.g., squared errors weighted by the spectrum errors),
+- and a minimizer,
+
+The minimizer is the method that adjusts the function
+parameters so that the model fits the data as closely as possible. The
+cost function defines the concept of how close a fit is to the
+data. See the general concept page on :ref:`Fitting <Fitting>` for a
+broader discussion of how these components interplay when fitting a
+model with Mantid.
+
+Several minimizers are included with Mantid and can be seleced in the
+`Fit Function property browser
+<http://www.mantidproject.org/MantidPlot:_Creating_Fit_Functions>`__
+or when using the algorithm :ref:`Fit <algm-Fit>` The following
+options are available:
+
+- `Simplex <https://en.wikipedia.org/wiki/Nelder%E2%80%93Mead_method>`__
+- `SteepestDescent <https://en.wikipedia.org/wiki/Gradient_descent>`__
+- `Conjugate gradient (Fletcher-Reeves imp.) <https://en.wikipedia.org/wiki/Nonlinear_conjugate_gradient_method>`__
+- `Conjugate gradient (Polak-Ribiere imp.) <https://en.wikipedia.org/wiki/Nonlinear_conjugate_gradient_method>`__
+- `BFGS (Broyden-Fletcher-Goldarfb-Shanno) <https://en.wikipedia.org/wiki/Broyden–Fletcher–Goldfarb–Shanno_algorithm>`__
+- `Levenberg-Marquardt <https://en.wikipedia.org/wiki/Levenberg-Marquardt_algorithm>`__ (default)
+- Levenberg-MarquardtMD
+- Damping
+- :ref:`FABADA <FABADA>`
+
+**TODO: Missing info: *Damping* differences between Lev-Mar and Lev-Mar "MD"**.
+
+All these algorithms are `iterative
+<https://en.wikipedia.org/wiki/Iterative_method>`__.  The *Simplex*
+algorithm, also known as Nelder–Mead method, belongs to the class of
+optimization algorithms without derivatives, or derivative-free
+optimization (another example of this class would be simulated
+annealing). Note that here simplex refers to downhill simplex
```

Comment:
I would suggest deleting "(another example of this class would be simulated annealing)" since it is referring to entirely different class of minimizers, i.e. a global minimizer.